tests/Core/File/GetMethodParametersTest.inc

```diff
@@ -45,6 +45,9 @@ function namespaceOperatorTypeHint(?namespace\Name $var1) {}
 /* testPHP8UnionTypesSimple */
 function unionTypeSimple(int|float $number, self|parent &...$obj) {}
 
+/* testPHP8UnionTypesWithSpreadOperatorAndReference */
+function globalFunctionWithSpreadAndReference(string|int ...$paramA, float|null &$paramB) {}
```

Comment:
This is normally invalid syntax. The reference would have to be listed first.

```php
php > function globalFunctionWithSpreadAndReference(string|int ...$paramA, float|null &$paramB) {}
PHP Fatal error:  Only the last parameter can be variadic in php shell code on line 1
```

Reply:
@zimzat Agreed. For context: the code in test case files for PHPCS doesn't actually have to be valid code. It just has to be code which allow us to test whether PHPCS handles certain syntax combinations correctly (in this case), and as this test confirms, it now does.

Reply:
FYI: fixed it up now either way ;-)